routes/web.php

```diff
@@ -11,7 +15,8 @@
 |
 */
 
-Route::get('/', 'UsersController@search');
+
+Route::get('/', [UsersController::class, 'search']);
```

Comment:
I believe you were missing the class imports for some controller classes.

I am getting this error:
```
Target class [UsersController] does not exist.
```

![screen_2023-07-31_022](https://github.com/nafiesl/silsilah/assets/8721551/b81279a9-0590-460f-8f00-4cca9c0e13c1)